client/darkweb.lua

```diff
@@ -34,8 +34,8 @@ end)
 -- Prolly a better minigame for this and needs a drilling anim
 local function breakCrate(entity)
     if haveItem('drill') then
-            exports['ps-ui']:Thermite(function(success)
-            if success then
+		exports['ps-ui']:Thermite(function(success)
```

Comment:
Instead of removing the indent you add another one?? I must be missing something here 😅

Reply:
i hate github's indent lol